src/modules/common/providers/UserContext.js

```diff
@@ -35,7 +35,11 @@ export const UserProvider = ({ children }) => {
                     )
 
                     if (response.data.user) {
-                        setUser(response.data.user)
+                        const user = {
```

Comment:
Make sure you merge in dev into your branch. The dev file now works fine so these changes to this file are not neccessary